session_security/middleware.py

```diff
@@ -51,7 +54,7 @@ def get_expire_seconds(self, request):
 
     def process_request(self, request):
         """ Update last activity time or logout. """
-        if not request.user.is_authenticated():
+        if not request.user.is_authenticated:
```

Comment:
Won't this always return True, even if the user is not authenticated, in Django < 2 ?